src/Components/Medicine/PrintPreview.tsx

```diff
@@ -150,31 +153,34 @@ const PrescriptionsTable = ({
   }
 
   if (!items.length) {
-    return;
+    return <div>No prescriptions available</div>; // Add a fallback message for empty items
   }
 
   return (
-    <table className="mb-8 mt-4 w-full border-collapse border-2 border-secondary-400">
-      <caption className="mb-2 caption-top text-lg font-bold">
-        {prn && "PRN"} Prescriptions
-      </caption>
-      <thead className="border-b-2 border-secondary-400 bg-secondary-50">
-        <tr>
-          <th className="max-w-52 p-1">Medicine</th>
-          <th className="p-1">Dosage</th>
-          <th className="p-1">Directions</th>
-          <th className="max-w-32 p-1">Notes / Instructions</th>
-        </tr>
-      </thead>
-      <tbody className="border-b-2 border-secondary-400">
-        {items.map((item) => (
-          <PrescriptionEntry key={item.id} obj={item} />
-        ))}
-      </tbody>
-    </table>
+    <div className="overflow-x-auto">
```

Comment:
Why was this made scrollable? Print preview content should not be scrollable within the paper view.